source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB specific information to users,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
```

Comment:
[Nit, minimalism]

Maybe "is knowledgeable of" > "accesses" ?

Reply:
I'm hesitant here only because the docs chatbot is trained on the docs but does not access it in real time.

Reply:
Got it. We can leave this as is!